src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -347,14 +349,14 @@ public static function provideInvalidInputValues()
     /**
      * @dataProvider provideInputValues
      */
-    public function testValidInput($expected, $value)
+    public function testValidInput(?string $expected, null|float|string $value)
     {
         $process = $this->getProcess('foo');
         $process->setInput($value);
         $this->assertSame($expected, $process->getInput());
     }
 
-    public static function provideInputValues()
+    public static function provideInputValues(): array
```

Comment:
How easy would it be for you to revert all the added return types from providers?
They don't provide any benefit IMHO but make reviewing harder + will generate merge needless conflicts.

Reply:
It's a manual process, but no problem